app/controllers/Api.scala

```diff
@@ -49,9 +49,16 @@ object Api extends Controller with PanDomainAuthActions {
 
   // can be hidden behind multiple auth endpoints
   private def getContentBlock[R] = { implicit req: R =>
-    val qs = req match {
-      case r: UserRequest[_] => r.queryString + ("email" -> Seq(r.user.email))
-      case r: Request[_] => r.queryString
+    val qs: Map[String, Seq[String]] = req match {
+      case r: UserRequest[_] =>
+        val query = r.queryString + ("email" -> Seq(r.user.email))
+
+        val supportAtoms = r.cookies.get("support-atoms").fold(false)(cookie => cookie.value == "1")
+        if(supportAtoms) query + ("supportAtoms" -> Seq(supportAtoms.toString)) else query
+
+      case r: Request[_] =>
+        val supportAtoms = r.cookies.get("support-atoms").fold(false)(cookie => cookie.value == "1")
+        if(supportAtoms) r.queryString + ("supportAtoms" -> Seq(supportAtoms.toString)) else r.queryString
     }
```

Comment:
The cookies logic can't be taken out of the pattern match because the request is of type `R` 😢 We can probably think of something to remove the `if`, but since this code is just temporary it might not worth it.